utility/src/test/TestManifest.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you can, please keep the name AndroidManifest. It’s conventional to use that name, so alternatives should only be used if necessary.

Reply:
Done!